src/server/visual-diff-reporter.js

```diff
@@ -101,7 +101,7 @@ function flattenResults(session, browserData, fileData) {
 				});
 			}
 			const testData = fileData.tests.get(testName);
-			const bytediff = !t.passed && !!info?.diff && info?.pixelsDiff === 0;
+			const bytediff = !t.passed && info?.diff === undefined && info?.pixelsDiff === 0;
```

Comment:
`info.diff` is going to be set to a string value of the path to the diff PNG. When `pixelsDiff` is `0` -- i.e. when there's a byte diff only, then `info.diff` will be `undefined`. `!!someString` will always yield `true` and `!!undefined` will yield `false`, which is the opposite of what we want.